smart-cards/src/main/java/gov/va/api/health/smartcards/patient/PatientController.java

```diff
@@ -181,6 +197,15 @@ private static void validateCredentialTypes(List<CredentialType> credentials) {
       throw new Exceptions.NotImplemented(
           String.format("Not yet implemented support for %s", requestedButUnimplemented));
     }
+    // Reject a request with ONLY #health-card
+    if (credentials.equals(List.of(CredentialType.HEALTH_CARD))) {
+      throw new Exceptions.BadRequest("Specify a more granular credential type");
+    }
+    // Reject a request with only #immunization
+    if (credentials.contains(CredentialType.IMMUNIZATION)
+        && !credentials.contains(CredentialType.COVID_19)) {
+      throw new Exceptions.NotImplemented("Only support COVID19 credential type");
```

Comment:
should this be `covid19` lowercase to match `https://smarthealth.cards#covid19`?